rpcq/_client.py

```diff
@@ -56,18 +56,21 @@ def __init__(self, endpoint: str, timeout: float = None):
         # Cache of replies so that different tasks can share results with each other
         self._replies: Dict[str, Union[RPCReply, RPCError]] = {}
 
-    async def call_async(self, method_name: str, *args, **kwargs):
+    async def call_async(self, method_name: str, *args, rpcq_timeout: float = None, **kwargs):
         """
         Send JSON RPC request to a backend socket and receive reply (asynchronously)
 
         :param method_name: Method name
         :param args: Args that will be passed to the remote function
+        :param float rpcq_timeout: Timeout in seconds for Server response, set to None to disable the timeout
         :param kwargs: Keyword args that will be passed to the remote function
         """
-        if self.timeout:
+        if rpcq_timeout is not None:
+            rpcq_timeout = self.timeout
```

Comment:
This is confusing me... If there is a non zero, non None argument passed, then overwrite it with the default? Wouldn't we want to opposite? Seems like `rpcq_timeout` is functioning as a bool to determine if the default timeout should be applied or not

Reply:
Good point looks like a typo